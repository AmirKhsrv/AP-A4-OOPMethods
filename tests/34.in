show_author_info 640
recommend_first_approach 112
show_author_info 698
best_book
best_reviewer
recommend_second_approach 107
recommend_first_approach 119
credit 13
