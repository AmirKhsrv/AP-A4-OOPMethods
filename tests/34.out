id: 640
Name: Amy Spalding
Year of Birth: 1965
Place of Birth: Sudan
Member Since: October 20 2007
Genres: Young Adult
Books:
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 698
Name: Sherry Gammon
Year of Birth: 1990
Place of Birth: India
Member Since: June 4 2013
Genres: Young Adult
Books:
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
Reviews:
id: 356 Rating: 3 Likes: 0 Date: August 24 2018
id: 396 Rating: 2 Likes: 558 Date: January 16 2011
id: 677 Rating: 5 Likes: 3 Date: August 13 2018
id: 914 Rating: 5 Likes: 0 Date: June 3 2019
id: 1654 Rating: 0 Likes: 0 Date: February 6 2019
id: 1665 Rating: 5 Likes: 0 Date: November 21 2017
id: 1866 Rating: 0 Likes: 0 Date: December 27 2018
id: 2173 Rating: 2 Likes: 0 Date: February 17 2012
id: 2285 Rating: 5 Likes: 0 Date: March 28 2013
id: 2901 Rating: 5 Likes: 0 Date: July 20 2017
id: 4481 Rating: 5 Likes: 2 Date: May 21 2016
id: 5827 Rating: 4 Likes: 0 Date: November 25 2019
id: 5975 Rating: 1 Likes: 0 Date: March 28 2018
id: 6899 Rating: 5 Likes: 1 Date: February 27 2018
id: 8914 Rating: 1 Likes: 0 Date: July 12 2013
id: 9248 Rating: 5 Likes: 0 Date: October 4 2018
id: 9999 Rating: 5 Likes: 0 Date: March 18 2015
id: 10317 Rating: 4 Likes: 0 Date: April 18 2014
id: 10521 Rating: 4 Likes: 0 Date: September 13 2019
id: 10822 Rating: 5 Likes: 0 Date: May 11 2016
id: 11689 Rating: 5 Likes: 1 Date: January 27 2017
id: 12274 Rating: 5 Likes: 0 Date: August 25 2015
id: 12379 Rating: 5 Likes: 0 Date: June 23 2015
id: 12431 Rating: 0 Likes: 1 Date: July 25 2019
id: 12626 Rating: 3 Likes: 0 Date: September 18 2019
id: 13444 Rating: 0 Likes: 0 Date: June 26 2013
id: 13813 Rating: 3 Likes: 0 Date: May 27 2018
id: 14083 Rating: 3 Likes: 0 Date: October 26 2014
id: 14142 Rating: 4 Likes: 0 Date: December 27 2014
id: 14199 Rating: 3 Likes: 4 Date: October 12 2015
id: 14427 Rating: 1 Likes: 2 Date: June 15 2018
id: 16431 Rating: 5 Likes: 0 Date: October 28 2014
id: 16509 Rating: 5 Likes: 0 Date: March 17 2016
id: 16740 Rating: 3 Likes: 0 Date: April 26 2016
id: 16829 Rating: 0 Likes: 0 Date: October 15 2017
id: 17340 Rating: 3 Likes: 0 Date: November 13 2017
0.007470
